### PR TITLE
Add UG tips, help and clear section

### PR DESCRIPTION
**Pull request by enhao25**
Updated with help and clear commands that is present in the app originally. 

Included some tips that I could think of.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -20,6 +21,7 @@ It allows for faster and more effective student management.
   * [Adding a tuition session: `add_session`](#adding-a-tuition-session-add_session)
   * [Deleting a tuition session: `delete_session`](#deleting-a-tuition-session-delete_session)
   * [Getting monthly fee for a particular student: `fee`](#getting-monthly-fee-for-a-particular-student-fee)
+  * [Clears all entries in the program: `exit`](#clearing-all-entries--clear)
```
Comment (JonahhGohh) [inline docs/UserGuide.md]: Should be "Clears all entries in the program: `clear`"

File: docs/UserGuide.md [hunk 2]
```diff
@@ -110,8 +128,15 @@ Deletes the specified student from TutorBuddy
 
 Format: `delete_student INDEX` <br>
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
```
Comment (JonahhGohh) [inline docs/UserGuide.md:202]: Is the "Tip:" meant to be bolded?
or is this the intended outcome:
image

**Review by JonahhGohh — COMMENTED** [review]
A few comments but should be mostly good.

File: docs/UserGuide.md (hunk 2 above)
Comment (enhao25) [reply docs/UserGuide.md:202]: Yeap should be correct. I copied over from https://github.com/nus-cs2103-AY2021S2/tp/blob/master/docs/UserGuide.md. On github, it shows as per the picture as well, however when its in the webpage, it should looks as what we expected.

Comment (JonahhGohh) [reply docs/UserGuide.md:202]: ohhh ok nice

**Review by JonahhGohh — APPROVED** [review]
LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (enhao25) [reply docs/UserGuide.md]: Updated.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -12,6 +12,7 @@ It allows for faster and more effective student management.
 **Table of Contents**
 * [Quick start](#quick-start)
 * [Features](#features)
+  * [Open help panel: `help`](#viewing-help--help)
```
Comment (samleewy) [inline docs/UserGuide.md:19]: Should help be placed below together with `clear` and `exit` instead?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -124,6 +136,12 @@ Adds a student to the TutorBuddy
 Format: `add_student n/NAME p/STUDENT_PHONE_NUMBER e/EMAIL a/ADDRESS l/STUDY_LEVEL g/GUARDIAN_PHONE_NUMBER r/RELATIONSHIP_WITH_GUARDIAN` <br>
 * `STUDENT_PHONE_NUMBER`, `GUARDIAN_PHONE_NUMBER` should be in Singapore's phone formatting (i.e. starting with either 6, 8 or 9 and 8 digits)
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+TutorBuddy only allows students with distinct names, however, its case sensitive. 
+Hence, students with the same name with different cases are treated as non-duplicated students. <br>
+(E.g. TutorBuddy allows students with both John Doe and john doe)
```
Comment (samleewy) [inline docs/UserGuide.md]: This is not related to your UG, but practicality wise, isn't it funny to have us accept same name but different cases? HAHA

File: docs/UserGuide.md [hunk 5]
```diff
@@ -198,6 +223,11 @@ Format: `add_session n/STUDENT_NAME d/DATE t/TIME k/DURATION s/SUBJECT f/FEE`
 * `DURATION` should be in minutes
 * `FEE` should be the total tuition fee for the total duration
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+TutorBuddy takes care of overlapping session for you so that you don't have to worry about it.<br>
+(Overlapping session will return an error)
```
Comment (samleewy) [inline docs/UserGuide.md]: Not too sure if the 'Overlapping session will return an error' is necessary. But the 'takes care of overlapping session' also sounds very ambiguous (like we have some algo or something).

Perhaps phrase it in a way like 'TutorBuddy takes care of overlapping session for you by giving a gentle prompt, so you don't have to worry about it'

File: docs/UserGuide.md [hunk 6]
```diff
@@ -227,6 +257,16 @@ Format: `fee n/STUDENT_NAME m/MONTH y/YEAR`
 Examples:
 * `fee n/John Lee m/1 y/2021` get John Lee monthly fee for January 2021
 
+### Clearing all entries : `clear`
+
+Clears all entries from the TutorBuddy application.
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+TutorBuddy gives you some "sample" data to work with at the start. <br>
```
Comment (samleewy) [inline docs/UserGuide.md]: I think don't need to quote 'sample' haha

**Review by samleewy — COMMENTED** [review]
Minor comments!

File: docs/UserGuide.md (hunk 3 above)
Comment (enhao25) [reply docs/UserGuide.md:19]: Hmmmm I followed https://nus-cs2103-ay2021s2.github.io/tp/UserGuide.html ah. I think the reason why they did that is that it is more sequential ah. A person who starts must know how to "get help" before even starting to learn about the commands. After trying out all the commands, he / she can then clear the application and then "start" actually using their own data for the application. Thereafter they can exit the application. This seems to be quite a logic flow. What do you think?

File: docs/UserGuide.md (hunk 4 above)
Comment (enhao25) [reply docs/UserGuide.md]: Haha yeah ah. But I think if we want to change, we need to update it at alot of places though. So I put it here as a "feature", so people can't dispute with us, saying its a bug ah.

File: docs/UserGuide.md (hunk 6 above)
Comment (enhao25) [reply docs/UserGuide.md]: Updated

File: docs/UserGuide.md (hunk 5 above)
Comment (enhao25) [reply docs/UserGuide.md]: Yeap sounds better! Have updated.

File: docs/UserGuide.md (hunk 4 above)
Comment (enhao25) [reply docs/UserGuide.md]: Anw if you want, we could create an issue, then if we have time then we can work on that ah. If not then we will just go along with this?

**Pull request merged**